imqsauth/http.go

```diff
@@ -597,14 +640,20 @@ func httpHandlerUpdateUser(central *ImqsCentral, w http.ResponseWriter, r *httpR
 	firstname := strings.TrimSpace(r.http.URL.Query().Get("firstname"))
 	lastname := strings.TrimSpace(r.http.URL.Query().Get("lastname"))
 	mobilenumber := strings.TrimSpace(r.http.URL.Query().Get("mobilenumber"))
+	strAuthusertype := strings.TrimSpace(r.http.URL.Query().Get("authusertype"))
+	iAuthUserType, err := strconv.Atoi(strAuthusertype)
+	authUserType := authaus.AuthUserType(iAuthUserType)
```

Comment:
This cast will not catch invalid values. Try the concept in a little test program.